controller/lib/comms/command_handlers-mock.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __COMMAND_HANDLERS_H
+#define __COMMAND_HANDLERS_H
+#include "network_protocol.pb.h"
+
+void command_handler(Command);
+void ack_handler(GuiAck);
+
+#endif
```

Comment:
this is just to trick platformio, otherwise it pulls in the real lib into test build, I will update tests to google test and see if this can be resolved